controls/cis_ubuntu2404.yml

```diff
@@ -1090,14 +1090,19 @@ controls:
     levels:
       - l1_server
       - l1_workstation
-    related_rules:
-      - file_at_deny_not_exist
+    rules:
+      - file_at_allow_exists
       - file_groupowner_at_allow
       - file_owner_at_allow
       - file_permissions_at_allow
-    status: planned
-    notes: TODO. Partial/incorrect implementation exists.See related rules. Analogous to ubuntu2204/5.1.9.
-
+      - file_at_deny_not_exist
```

Comment:
`/etc/at.deny` does not need to exist, but _can_ exist as long as the permissions are ok. I don't think the `file_at_deny_not_exist` should be added to the rules list.

Reply:
Done